modules/cnv_importing_vm_disk_DVs.adoc

```diff
@@ -1,10 +1,16 @@
 [[importvm-datavolumes]]
 === Cloning an existing virtual machine image with DataVolumes
```

Comment:
Just noticed that above you say cloning but what we are talking about here is importing...

Reply:
Aha, good catch. No wonder I'm getting all these mixed up. Fixed in:https://github.com/openshift/openshift-docs/pull/12518/commits/4e71a10ddff35d1d0d230ce8471c81b8992919c4